modules/cluster-logging-rn-5.5.3.adoc

```diff
@@ -0,0 +1,36 @@
+//module included in cluster-logging-release-notes.adoc
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-5-3_{context}"]
+= Logging 5.5.3
+This release includes link:https://access.redhat.com/errata/RHBA-2022:6858[OpenShift Logging Bug Fix Release 5.5.3].
+
+[id="openshift-logging-5-5-3-bug-fixes_{context}"]
+== Bug fixes
+* Before this update log entries that had structured messages included the original message field which made the entry larger. This update removes the message field for structured logs to reduce the increased size. (link:https://issues.redhat.com/browse/LOG-2759[LOG-2759])
+
+* Before this update, the collector configuration excluded logs from `collector`, `default-log-store`, and `visualization`  pods, but was unable to exclude logs archived in a `.gz` file. With this update, archived logs stored as `.gz` files of `collector`, `default-log-store`, and `visualization` pods are also excluded. (link:https://issues.redhat.com/browse/LOG-2844[LOG-2844])
+
+* Before this update, when requests to an unavailable pod was sent through the gateway, no alert would fire to warn of the disruption. With this update, the individual alerts will fire if the gateway has issues completing a write or read request respectively. (link:https://issues.redhat.com/browse/LOG-2884[LOG-2884])
+
+* Before this update, pod metadata could be altered by fluent plugins because the values passed through the pipeline by reference. This update fixes that to ensure each log message receives a copy of the pod metadata so each message processes independently. (link:https://issues.redhat.com/browse/LOG-3046[LOG-3046])
```

Comment:
I'm not sure what `fixes that` refers to. Does `that` refer to the pod metadata, the values, the pipeline references? 

What do you think of something like this?

`This update fixes how pod metadata is referenced by ensuring each log message receives a copy of the pod metadata, so now each message independently processes its reference.` 

Note, I wasn't sure what the messages are processing. I inferred that it was processing its reference. I might be incorrect.

Reply:
"This update ensures each log message receives a copy of the pod metadata so each message processes independently."